### PR TITLE
Added BubbleSort algorithm in csharp

### LINKED ISSUE #485: Bubble Sort in C#
This is an open issue for the Bubble Sort algorithm implementation in C#. If you wish to contribute, just make some noise and we will assign you.

### PR DESCRIPTION
**Pull request by forsas**
**Hi!**  
Thanks for considering for contributing to this ever-growing list of algorithm and data structure implementations.
Your contribution is valuable. Use the follow template for creating PRs. (Delete this part)
---

**Fixes #485**

By submitting this pull request I confirm I've read and complied with the below declarations.

-   [x] I have read the [Contribution guidelines](https://github.com/FOSS-UCSC/FOSSALGO/blob/master/CONTRIBUTING.md) 
and I am confident that my PR reflects them.

-   [x] I have followed the [coding guidelines](https://github.com/FOSS-UCSC/FOSSALGO/blob/master/CONTRIBUTING.md#cs) 
for this project.

-   [x] My code follows the [skeleton code structure](https://github.com/FOSS-UCSC/FOSSALGO/blob/master/CONTRIBUTING.md#sample).

-   [x] This pull request has a descriptive title. 
For example, `Added {Algorithm/DS name} [{Language}]`, not `Update README.md` or `Added new code`.

-   [x] This pull request shall only be merged once the build passes.

-   [x] I have mentioned the issue number correctly (with hyper-link) in this pull request description.

### CODE REVIEW

File: algorithms/ar-bsrt/csharp/BubbleSort.cs [hunk 1]
```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Diagnostics;
+
+namespace BubbleSort
+{
+    public class BubbleSort
```
Comment (RavinduSachintha) [inline algorithms/ar-bsrt/csharp/BubbleSort.cs:6]: Codacy found an issue: [Add a 'protected' constructor or the 'static' keyword to the class declaration.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6326096)

File: algorithms/ar-bsrt/csharp/BubbleSort.cs [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+            return array;
+        }
+
+        public static void Main()
+        {
+            var array = new int[] { 2, 3, 0, 4, 1, 9, 6, 8, 5, 7 };
+
+            Console.WriteLine("Initial array:");
+            foreach (var element in array)
```
Comment (RavinduSachintha) [inline algorithms/ar-bsrt/csharp/BubbleSort.cs:32]: Codacy found an issue: [Add curly braces around the nested statement(s) in this 'foreach' block.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6326096)

File: algorithms/ar-bsrt/csharp/BubbleSort.cs [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+                Console.Write(element + " ");
+
+            var stopwatch = new Stopwatch();
+            stopwatch.Start();
+
+            var sortedArray = GetBubbleSortedArray(array);
+
+            stopwatch.Stop();
+
+            Console.WriteLine(Environment.NewLine + "Sorted array:");
```
Comment (RavinduSachintha) [inline algorithms/ar-bsrt/csharp/BubbleSort.cs:42]: Codacy found an issue: [Remove this logging statement.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6326096)

File: algorithms/ar-bsrt/csharp/BubbleSort.cs [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+            return array;
+        }
+
+        public static void Main()
+        {
+            var array = new int[] { 2, 3, 0, 4, 1, 9, 6, 8, 5, 7 };
+
+            Console.WriteLine("Initial array:");
+            foreach (var element in array)
+                Console.Write(element + " ");
```
Comment (RavinduSachintha) [inline algorithms/ar-bsrt/csharp/BubbleSort.cs:33]: Codacy found an issue: [Remove this logging statement.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6326096)

File: algorithms/ar-bsrt/csharp/BubbleSort.cs [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+            }
+
+            return array;
+        }
+
+        public static void Main()
+        {
+            var array = new int[] { 2, 3, 0, 4, 1, 9, 6, 8, 5, 7 };
+
+            Console.WriteLine("Initial array:");
```
Comment (RavinduSachintha) [inline algorithms/ar-bsrt/csharp/BubbleSort.cs:31]: Codacy found an issue: [Remove this logging statement.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6326096)

File: algorithms/ar-bsrt/csharp/BubbleSort.cs [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+
+            var stopwatch = new Stopwatch();
+            stopwatch.Start();
+
+            var sortedArray = GetBubbleSortedArray(array);
+
+            stopwatch.Stop();
+
+            Console.WriteLine(Environment.NewLine + "Sorted array:");
+            foreach (var element in sortedArray)
```
Comment (RavinduSachintha) [inline algorithms/ar-bsrt/csharp/BubbleSort.cs:43]: Codacy found an issue: [Add curly braces around the nested statement(s) in this 'foreach' block.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6326096)

File: algorithms/ar-bsrt/csharp/BubbleSort.cs [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+                    }
+                }
+            }
+
+            return array;
+        }
+
+        public static void Main()
+        {
+            var array = new int[] { 2, 3, 0, 4, 1, 9, 6, 8, 5, 7 };
```
Comment (RavinduSachintha) [inline algorithms/ar-bsrt/csharp/BubbleSort.cs:29]: Codacy found an issue: [Remove the array type; it is redundant.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6326096)

File: algorithms/ar-bsrt/csharp/BubbleSort.cs (hunk 6 above)
Comment (RavinduSachintha) [inline algorithms/ar-bsrt/csharp/BubbleSort.cs:43]: Codacy found an issue: [Add curly braces around the nested statement(s) in this 'foreach' block.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6326096)

File: algorithms/ar-bsrt/csharp/BubbleSort.cs (hunk 1 above)
Comment (RavinduSachintha) [inline algorithms/ar-bsrt/csharp/BubbleSort.cs:6]: Codacy found an issue: [Add a 'protected' constructor or the 'static' keyword to the class declaration.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6326096)

File: algorithms/ar-bsrt/csharp/BubbleSort.cs (hunk 5 above)
Comment (RavinduSachintha) [inline algorithms/ar-bsrt/csharp/BubbleSort.cs:31]: Codacy found an issue: [Remove this logging statement.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6326096)

File: algorithms/ar-bsrt/csharp/BubbleSort.cs (hunk 2 above)
Comment (RavinduSachintha) [inline algorithms/ar-bsrt/csharp/BubbleSort.cs:32]: Codacy found an issue: [Add curly braces around the nested statement(s) in this 'foreach' block.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6326096)

File: algorithms/ar-bsrt/csharp/BubbleSort.cs (hunk 3 above)
Comment (RavinduSachintha) [inline algorithms/ar-bsrt/csharp/BubbleSort.cs:42]: Codacy found an issue: [Remove this logging statement.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6326096)

File: algorithms/ar-bsrt/csharp/BubbleSort.cs (hunk 7 above)
Comment (RavinduSachintha) [inline algorithms/ar-bsrt/csharp/BubbleSort.cs:29]: Codacy found an issue: [Remove the array type; it is redundant.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6326096)

File: algorithms/ar-bsrt/csharp/BubbleSort.cs (hunk 4 above)
Comment (RavinduSachintha) [inline algorithms/ar-bsrt/csharp/BubbleSort.cs:33]: Codacy found an issue: [Remove this logging statement.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6326096)

File: algorithms/ar-bsrt/csharp/BubbleSort.cs [hunk 8]
```diff
@@ -0,0 +1,53 @@
...
+            }
+
+            return array;
+        }
+
+        public static void Main()
+        {
+            var array = new [] { 2, 3, 0, 4, 1, 9, 6, 8, 5, 7 };
+
+            //Console.WriteLine("Initial array:");
```
Comment (RavinduSachintha) [inline algorithms/ar-bsrt/csharp/BubbleSort.cs:31]: Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6326096)

File: algorithms/ar-bsrt/csharp/BubbleSort.cs [hunk 9]
```diff
@@ -0,0 +1,53 @@
...
+            //{
+                //Console.Write(element + " ");
+            //}
+
+            //var stopwatch = new Stopwatch();
+            //stopwatch.Start();
+
+            var sortedArray = GetBubbleSortedArray(array);
+
+            //stopwatch.Stop();
```
Comment (RavinduSachintha) [inline algorithms/ar-bsrt/csharp/BubbleSort.cs:42]: Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6326096)

File: algorithms/ar-bsrt/csharp/BubbleSort.cs [hunk 10]
```diff
@@ -0,0 +1,53 @@
...
+            //Console.WriteLine("Initial array:");
+            //foreach (var element in array)
+            //{
+                //Console.Write(element + " ");
+            //}
+
+            //var stopwatch = new Stopwatch();
+            //stopwatch.Start();
+
+            var sortedArray = GetBubbleSortedArray(array);
```
Comment (RavinduSachintha) [inline algorithms/ar-bsrt/csharp/BubbleSort.cs:40]: Codacy found an issue: [Remove the unused local variable 'sortedArray'.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6326096)

**Review by Danushka96 — APPROVED** [review]

**Comment by forsas** [comment]
https://github.com/FOSS-UCSC/FOSSALGO/pull/595

**Pull request closed without merging**